client-no-storybook/src/components/TaskAppContainer.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import PropTypes from "prop-types";
+import { Flex, EditableHeading } from "monday-ui-react-core";
+
+import "../css/tasksAppContainer.css";
+import "monday-ui-react-core/dist/main.css";
+
+import { UserInput } from "./UserInput";
+import { TaskList } from "./TaskList";
+
+import { getAllTasks, addTask, deleteTask, updateTask } from "../redux/task.slice";
+
+export const TaskAppContainer = ({}) => {
+    const dispatch = useDispatch();
+
+    const {read, create, update} = useSelector((state) => state.tasks);
```

Comment:
the selector should be of entities or data relevant to the running of the app.
for example. in state.tasks I would expect to have a list of the tasks and their details.
You could have another branch in the state named appStatus or tasksStatus which you could save in if the app in in loading mode